### DISCUSSION TITLE
Error Trying to read a blob in Azure

### DISCUSSION THREAD

**Discussion by palominoc** [duckdb/duckdb#9266] [Q&A]
Hi!
I'm Trying to read a .parquet file in Azure Blob Storage, the funniest part is that the same code works on local, but when i deploy the function to Azure Functions appears this Error:

Error: Invalid Error: Fail to get a new connection for: https://{account}.blob.core.windows.net. Problem with the SSL CA cert (path? access rights?)

and the related code is this:

def procesar_datos(file_name):
    # DuckDB conection
    con = duckdb.connect()
    con.execute("INSTALL azure;")
    con.execute("LOAD azure;")
    con.execute(f"SET azure_storage_connection_string = '{account_conection}'")

    # Query to extract the info
    fecha_query = f"""
    SELECT DISTINCT CONCAT(EXTRACT(YEAR FROM fecha_alta), LPAD(EXTRACT(MONTH FROM fecha_alta), 2, '0')) as fecha
    FROM 'azure://oppm/{file_name}'
    """

I hope someone can help me with this. than for all!

**Comment by Mause** [comment]
Duplicate of https://github.com/duckdblabs/duckdb_azure/issues/8
